plugins/chat/assets/javascripts/discourse/services/chat-channels-manager.js

```diff
@@ -131,6 +131,16 @@ export default class ChatChannelsManager extends Service {
       .sort((a, b) => a?.slug?.localeCompare?.(b?.slug));
   }
 
+  @cached
+  get publicMessageChannelsWithActivity() {
```

Comment:
Same question here about urgent badge counts, at the moment we don't count mentions in DMs (defaults to 0 but being added soon) but we do track unread watched thread counts.